ReactQt/runtime/src/reactitem.cpp

```diff
@@ -558,8 +558,9 @@ void ReactItem::paint(QPainter* painter) {
 }
 
 void ReactItem::updatePolish() {
-    ReactFlexLayout::get(this)->setWidth(width());
-    ReactFlexLayout::get(this)->setHeight(height());
-
-    ReactFlexLayout::polish(this);
+    if (childItems().count() > 0) {
+        QQuickItem* view = childItems().at(0);
```

Comment:
This method should be moved to RootView, since we recalculate layout from top node. RootView usually have only one child, I'll add assert to check this.